specs/credentialmanagement/index.src.html

```diff
@@ -329,9 +331,16 @@ <h4 id="examples-post-signin">Post-sign-in Confirmation</h4>
         credential.send("https://example.com/sign-in/")
           .then(function (response) {
             if (response.status == 200)
+              // Inspect response to determine if this was a successful login then...
```

Comment:
Nit: Put this before the `if`, please. Otherwise you'll need `{}` around the clause for clarity (because it's multi-line).